### PR TITLE
#722 - Create a separate class for test container

### PR DESCRIPTION
**Pull request by genryxy**
Part of #722
Extract to a separate class creating a test container. Also use path `tmp` instead of `subir` in `MavenITCase`.

### CODE REVIEW

File: src/test/java/com/artipie/test/TestContainer.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+     * @param path Path for binding file system
+     * @param port Port for exposing
+     */
+    @SuppressWarnings("PMD.ConstructorOnlyInitializesOrCallOtherConstructors")
+    public TestContainer(final String image, final Path path, final int port) {
+        this.cntn = new GenericContainer<>(image)
+            .withCommand("tail", "-f", "/dev/null")
+            .withWorkingDirectory("/home/")
+            .withFileSystemBind(path.toString(), "/home");
+        Testcontainers.exposeHostPorts(port);
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/test/TestContainer.java:55]: @genryxy I'd suggest moving this command to `start()` method, it seems more logical and we will not need any suppressions here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@genryxy please check one comment

**Comment by genryxy** [comment]
@olenagerasimova thanks, corrected

**Comment by olenagerasimova** [comment]
@genryxy thanks
@olegmoz looks good to merge

**Review by olegmoz — APPROVED** [review]

**Comment by olegmoz** [comment]
@olenagerasimova please approve the PR

**Pull request merged**